defs/scenarios/storage/ceph/ceph-mon/mon_db_too_big.yaml

```diff
@@ -0,0 +1,21 @@
+checks:
+  mon_large_db:
+    input:
+      command: ceph_health_detail_json_decoded
+    expr: '.*mon .+ is using a lot of disk space.*'
+  health_warning:
+    property:
+      path: hotsos.core.plugins.storage.ceph.CephCluster.health_status
+      ops: [[eq, HEALTH_WARN]]
+conclusions:
+  db_too_large:
+    decision:
+      - mon_large_db
+      - health_warning
+    raises:
+      type: CephMonWarning
+      message: >-
```

Comment:
Maybe slight reworded:

ceph is reporting that the mon leveldb database used to store cluster metadata is using a lot of disk space which might be causing slow queries and delayed response to clients. Recommendation is to run compaction on the mon db.

Some detail on how to run compaction would be nice too. Also is there a way to tell which mon is the one affected or will be all of them?

Reply:
I've made the changes to print the mon name now. (used search-result-format-groups). If I also print the various approaches to compacting the leveldb, that makes the hotsos console output rather large.. I'm considering simply including a link to the knowledge article that Emre is working on, once its complete. Thoughts?

Reply:
ack, I will link the upstream doc.